okex-client/tests/client.rs

```diff
@@ -4,32 +4,33 @@ use rust_decimal::Decimal;
 use rust_decimal_macros::dec;
 
 use okex_client::{
-    OkexClient, OkexClientConfig, OkexClientError, OKEX_MINIMUM_WITHDRAWAL_AMOUNT,
-    OKEX_MINIMUM_WITHDRAWAL_FEE,
+    OkexClient, OkexClientConfig, OkexClientError, OkexInstrumentId, OkexMarginMode, OkexOrderSide,
+    OkexPositionSide, OKEX_MINIMUM_WITHDRAWAL_AMOUNT, OKEX_MINIMUM_WITHDRAWAL_FEE,
 };
 
 fn configured_okex_client() -> OkexClient {
     let api_key = env::var("OKEX_API_KEY").expect("OKEX_API_KEY not set");
     let passphrase = env::var("OKEX_PASSPHRASE").expect("OKEX_PASS_PHRASE not set");
     let secret_key = env::var("OKEX_SECRET_KEY").expect("OKEX_SECRET_KEY not set");
+    let simulated = env::var("OKEX_SIMULATED_TRADING").expect("OKEX_SIMULATED_TRADING not set");
     OkexClient::new(OkexClientConfig {
         api_key,
         passphrase,
         secret_key,
+        simulated,
     })
 }
 
 fn demo_okex_client() -> OkexClient {
-    // Set environment variable for "x-simulated-trading"
-    env::set_var("OKEX_SIMULATED_TRADING", "1");
-
     let api_key = env::var("OKEX_DEMO_API_KEY").expect("OKEX_DEMO_API_KEY not set");
     let passphrase = env::var("OKEX_DEMO_PASSPHRASE").expect("OKEX_DEMO_PASSPHRASE not set");
     let secret_key = env::var("OKEX_DEMO_SECRET_KEY").expect("OKEX_DEMO_SECRET_KEY not set");
+    let simulated = env::var("OKEX_SIMULATED_TRADING").expect("OKEX_SIMULATED_TRADING not set");
```

Comment:
Do we ever want to run the tests unsimulated? Why are we reading this setting from env and not hard coding it in the test?

Reply:
For majority of the tests, we can run them simulated against the demo API but not against endpoints to deposit or withdraw. Having removed the withdraw test, there is no need to run tests unsimulated. 

With regards to reading variables from the environment, I could easily hardcode them in the test but then the tests will be linked to my account's API keys. If I, for whatever reason, delete those keys, all tests will fail. If the demo API keys for galoy's OKX account are shared with me, then reading from the environment would not be needed.